### ISSUE TITLE
from_tag/to_tag + multiple repos is broken

### ISSUE THREAD

**Issue by mnagel** [ishepard/pydriller#116]
**Describe the bug**
I want to analyse multiple repos and I want to anyalze between tags (present in all repos).

**To Reproduce**
```
import re

from pydriller import RepositoryMining

repos = [
    "C:/jcb/code/montagel",
    "C:/jcb/code/Xrf",
]
print(repos)

starttag="com.jcbachmann.xrf.suite/v11"
endtag="com.jcbachmann.xrf.suite/v12"

for commit in RepositoryMining(path_to_repo=repos[0], from_tag=starttag, to_tag=endtag).traverse_commits():
    pass
print("repo 0 OK")

for commit in RepositoryMining(path_to_repo=repos[1], from_tag=starttag, to_tag=endtag).traverse_commits():
    pass
print("repo 1 OK")

for commit in RepositoryMining(path_to_repo=repos).traverse_commits():
    pass
print("repos untagged OK")

for commit in RepositoryMining(path_to_repo=repos, from_tag=starttag, to_tag=endtag).traverse_commits():
    pass
print("repos tagged OK")
```

```
C:\jcb\venvs\Xrf\Scripts\python.exe C:/jcb/code/Xrf-Python/Documentation/chapters/software/bug.py
['C:/jcb/code/montagel', 'C:/jcb/code/Xrf']
repo 0 OK
repo 1 OK
repos untagged OK
Traceback (most recent call last):
  File "C:/jcb/code/Xrf-Python/Documentation/chapters/software/bug.py", line 26, in <module>
    for commit in RepositoryMining(path_to_repo=repos, from_tag=starttag, to_tag=endtag).traverse_commits():
  File "C:\jcb\venvs\Xrf\lib\site-packages\pydriller\repository_mining.py", line 168, in traverse_commits
    self._conf.sanity_check_filters()
  File "C:\jcb\venvs\Xrf\lib\site-packages\pydriller\utils\conf.py", line 71, in sanity_check_filters
    self._check_correct_filters_order()
  File "C:\jcb\venvs\Xrf\lib\site-packages\pydriller\utils\conf.py", line 110, in _check_correct_filters_order
    self.get('git_repo').get_commit(self.get('from_commit')),
  File "C:\jcb\venvs\Xrf\lib\site-packages\pydriller\git_repository.py", line 144, in get_commit
    gp_commit = self.repo.commit(commit_id)
  File "C:\jcb\venvs\Xrf\lib\site-packages\git\repo\base.py", line 480, in commit
    return self.rev_parse(str(rev) + "^0")
  File "C:\jcb\venvs\Xrf\lib\site-packages\git\repo\fun.py", line 213, in rev_parse
    obj = name_to_object(repo, rev[:start])
  File "C:\jcb\venvs\Xrf\lib\site-packages\git\repo\fun.py", line 147, in name_to_object
    raise BadName(name)
gitdb.exc.BadName: Ref '['' did not resolve to an object

Process finished with exit code 1
```

**OS Version:**
Python 3.7 on Windows

**Comment by ishepard** [comment]
Hi @mnagel!
Thank you for catching this.
I am indeed able to reproduce the error. It only appears when more than 1 repo is given in input..I will look into it!
Thanks!

**Comment by ishepard** [comment]
I found the error, and I will work on a fix today 👍 thanks!

**Comment by mnagel** [comment]
Hi Davide,

thank you for the fast reaction. I am looking forward to the fix!

Best Regards
Michael

**Comment by ishepard** [comment]
I just released version 1.15 with the fix!
Thank you again for catching the bug.
Cheers!

**Comment by mnagel** [comment]
Can confirm the fix! Thank you!
